scripts/sendpayment.py

```diff
@@ -75,7 +77,15 @@ def main():
                 sys.exit(EXIT_ARGERROR)
             destaddr = parsed['address']
             if 'jmnick' in parsed:
+                if "pj" in parsed:
+                    parser.error("Cannot specify both BIP79 and Joinmarket "
+                                 "peer-to-peer payjoin at the same time!")
```

Comment:
Hmm, interesting.
If we add to JM the receiver behaviour for BIP79++, then I presume the `pj=` setting will be a hidden service onion link. So are you saying - we could have it so that either is possible, i.e. one is a fallback to the other?
It's certainly possible but I do feel it's getting too complicated at that point. My feeling would be, if we can see the case for allowing both later, let's add that to the client code. But since the client can't currently do that, then this check can be removed at the point in the future when it can.

Reply:
I just don't see the need for this check, could just ignore `jmnick=` if `pj=` is given (as BIP79++ is a lot faster than JM P2P payjoin).

Reply:
A user that gives a bip21uri with both parameters set has done something wrong, copied incorrectly, got confused somehow. I think it should be an error condition because we should always err on the side of caution.
I know that it's acceptable to have unrecognized BIP21 parameters according to the rules, but this is different to me, as it is very strong evidence of a mistake by the user. Any evidence of user confusion should be a "hard stop".

Reply:
Ok, probably you're right on this.